### PR TITLE
Remove note for concurrency primitives from readme

### PR DESCRIPTION
**Pull request by puzpuzpuz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,14 +120,12 @@ Refer to [the documentation](DOCUMENTATION.md) to learn more about supported con
 * Additional data structures and simple messaging constructs such as **Set**, **MultiMap**, **Queue**, **Topic**
 * Cluster-wide unique ID generator, called **FlakeIdGenerator**
 * Distributed, CRDT based counter, called **PNCounter**
-* Primitives for distributed computing such as **Lock**, **Semaphore**, **Atomic Long** *
+* Distributed concurrency primitives from CP Subsystem such as **Lock**, **Semaphore**, **AtomicLong**
```
Comment (mdumandag) [inline README.md:123]: Should we also add `CountDownLatch` and `AtomicReference`?

Comment (puzpuzpuz) [reply README.md:123]: I've kept the list short enough, as we don't list all DSes here (say, `ReplicatedMap` is not mentioned). Anyone who is interested in CP data structures will find those in the full documentation. WDYT?

Comment (mdumandag) [reply README.md:123]: Oh, okay. Then, we can list all of them here. https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/DOCUMENTATION.md#2-features . I see that there is no mention of CP subsystem structures there

Comment (puzpuzpuz) [reply README.md]: Good suggestion. Added CP DSes into the list in 6d6102e35d6f44c774779786d0c986077075c153

**Review by mdumandag — APPROVED** [review]

**Pull request merged**